servers/display_server.h

```diff
@@ -292,6 +292,14 @@ class DisplayServer : public Object {
 	virtual TypedArray<Rect2> get_display_cutouts() const { return TypedArray<Rect2>(); }
 	virtual Rect2i get_display_safe_area() const { return screen_get_usable_rect(); }
 
+	enum InitialScreen {
+		INITIAL_SCREEN_AUTO = -5, // Remembers last screen position.
+		INITIAL_SCREEN_WITH_MOUSE_FOCUS = -4,
+		INITIAL_SCREEN_WITH_KEYBOARD_FOCUS = -3,
+		INITIAL_SCREEN_PRIMARY = -2,
+		INITIAL_SCREEN_OF_MAIN_WINDOW = -1,
```

Comment:
You can use the other enum to copy values, e.g.
```suggestion
		INITIAL_SCREEN_OF_MAIN_WINDOW = SCREEN_OF_MAIN_WINDOW,
```